backstory/src/demo/webapp/perspective-demo.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Perspective</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+    <script src="perspective.js"></script>
+  </head>
+
+  <body class="vertical-flex-container" id="perspective-page">
+    <div class="content vertical-flex-container" id="perspective-content">
+      <h1 class="header blue-header">Perspective Demo</h1>
+
+      <div class="vertical-flex-container" id="perspective-div">
+        <textarea id="text-for-analysis" name="text"></textarea>
+        <button type="button" onclick="displayScores()">Display Scores</button>
+      </div>
+
+      <div id="attributes"></div>
+    </div>
+  </body>
+</html>
```

Comment:
``ac-basics`` was just merged into master, so it should be based on master either way now. However, I don't believe it would have removed these differences as the HTML code from that branch evolved slightly as I actually crafted the back end for the demo e.g. I re-ided the ``toxicity-percentage`` div to ``attributes`` because I began displaying more than just toxicity.

Reply:
Done